doc/news/changes.h

```diff
@@ -269,8 +269,9 @@ inconvenience this causes.
  (Bruno Turcksin, 2016/05/22)
   </li>
 
- <li> Move FESystem's auxiliary functions to FETools and extend them to cover
- a non-tensor product case used in the enriched finite element.
+ <li> New: There are now additional functions in namespace FETools that build
+ finite elements out of simpler finite elements, either by forming tensor
+ product or by enriching the set of shape functions.
```

Comment:
of course, as discussed here https://github.com/dealii/dealii/issues/2628, usage of those functions outside of the context of partition-of-unity method may lead to linearly dependent shape functions, especially if multiplicities are not equal 1.